src/test/addresses_deployers.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+//
+// Copyright (C) 2021 Dai Foundation
```

Comment:
consider adding license copyright and description to `addresses_mainnet.sol` as well.